vcpkg.json

```diff
@@ -61,6 +67,7 @@
             "description": "Build OpenCL backend",
             "dependencies": [
                 "boost-compute",
+                "boost-program-options",
```

Comment:
I don't think this is a good dependency for the project. It is only required for some of the tests in the clFFT project and those aren't built. I think the vcpkg should minimize the number of dependencies installed whenever possible.

Reply:
OK.
I get it out, and resend the PR.

Reply:
Or should I move it to the openblasfftw section ?

Reply:
OK, done